bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRIssueBot.java

```diff
@@ -75,18 +74,22 @@ public String toString() {
     public List<WorkItem> getPeriodicItems() {
         var issues = poller.updatedIssues();
         var items = issues.stream()
-                .map(i -> (WorkItem) new IssueWorkItem(this, i, e -> poller.retryIssue(i)))
+                .map(i -> (WorkItem) new CSRIssueWorkItem(this, i, e -> poller.retryIssue(i)))
                 .toList();
         poller.lastBatchHandled();
         return items;
     }
 
     @Override
     public String name() {
-        return CSRBotFactory.NAME;
+        return PullRequestBotFactory.NAME;
```

Comment:
The name of this `Bot` needs to be unique to avoid creating confusion with log messages. Perhaps add `-csr` as a suffix.